src/video/canvas.cpp

```diff
@@ -121,14 +124,11 @@ Canvas::draw_surface(const SurfacePtr& surface,
      position.y + static_cast<float>(surface->get_height()) < cliprect.get_top())
     return;
 
-  auto request = new(m_obst) TextureRequest();
```

Comment:
What??

Reply:
Sorry. What is this syntax? `new(m_obst) TextureRequest()`

Reply:
Creating pointer with `obstack`.